[ews-build] Improve error message when Merge-Queue fails to find reviewer

aj062 opened the pull request:
#### fd260f7fe6d34af0c191a796c773bebf706307fe
<pre>
[ews-build] Improve error message when Merge-Queue fails to find reviewer
<a href="https://bugs.webkit.org/show_bug.cgi?id=243470">https://bugs.webkit.org/show_bug.cgi?id=243470</a>

Reviewed by Ryan Haddad.

* Tools/CISupport/ews-build/steps.py:
(ValidateCommitterAndReviewer.start):
(ValidateCommitMessage.run):
* Tools/CISupport/ews-build/steps_unittest.py:

Canonical link: <a href="https://commits.webkit.org/253083@main">https://commits.webkit.org/253083@main</a>
</pre>

ryanhaddad commented on Tools/CISupport/ews-build/steps.py:
```diff
@@ -1770,7 +1770,8 @@ def start(self):
             # Change has not been reviewed in bug tracker. This is acceptable, since the ChangeLog might have 'Reviewed by' in it.
-            self.descriptionDone = 'Validated committer'
+            self._addToLog('stdio', f'Reviewer not found. Commit message  will be checked for reviewer name in later steps\n')
+            self.descriptionDone = 'Validated committer, Reviewer not found'
```
Nit: since commiter is lowercase, let's make reviewer lowercase too.

aj062 replied:
Will do.

aj062 committed:
[ews-build] Improve error message when Merge-Queue fails to find reviewer
https://bugs.webkit.org/show_bug.cgi?id=243470

Reviewed by Ryan Haddad.

* Tools/CISupport/ews-build/steps.py:
(ValidateCommitterAndReviewer.start):
(ValidateCommitMessage.run):
* Tools/CISupport/ews-build/steps_unittest.py:

Canonical link: https://commits.webkit.org/253083@main
